### PR TITLE
added steps for read_only and bulk access roles

### PR DESCRIPTION
**Pull request by ashwinkumar12345**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/security-access-control/users-roles.md [hunk 1]
```diff
@@ -117,3 +117,62 @@ Role | Description
...
+1. Next to `kibana_user`, choose the **Edit** button.
+- If you don't already have a `kibana_user` role, choose the *+* button and create the role.
+1. In the **Users** field, add your `read_only_index` role and choose **Submit**.
+1. Next to `kibana_read_only`, choose the **Edit** button.
+- If you don't already have a `kibana_read_only` role, choose the *+* button and create the role.
+1. In the **Users** field, add your `read_only_index` role and choose **Submit**.
+
+Create a new tenant and grant role access:
+
+1. Choose **Security**, **Tenants**, enter your desired name.
```
Comment (dinusX) [inline docs/security-access-control/users-roles.md]: It's mentioned twice to enter your desired name.

File: docs/security-access-control/users-roles.md [hunk 2]
```diff
@@ -117,3 +117,62 @@ Role | Description
...
+1. Choose the *+* button, enter your desired tenant name.
+1. Choose **Submit**.
+1. Choose **Security**, **Roles**.
+1. Next to the `read_only_index` role, choose **Edit**.
+1. Choose the **Tenants Permissions** tab, in **Permissions**, select `kibana_all_read`.
+1. Add a tenant pattern to select all required tenants.
+1. Choose **Save Role Definition**.
```
Comment (dinusX) [inline docs/security-access-control/users-roles.md]: Shouldn't this be the last step?

Comment (ashwinkumar12345) [reply docs/security-access-control/users-roles.md]: Yes, removed the last step.

File: docs/security-access-control/users-roles.md (hunk 1 above)
Comment (ashwinkumar12345) [reply docs/security-access-control/users-roles.md]: fixed.

**Pull request merged**